app/learning.js

```diff
@@ -4,39 +4,30 @@
  * version 1.1
  *
  * Description:
- * Main script for initiating the app. Constants and variables are loaded
- * in from config.js and the functions from functions.js.
+ * Main script for initiating the learning app.
  */
 
-define(function (require) {
-  requirejs(['jquery'], function( $ ) {
-    // Include message functions
-    var messages = require('./messages');
-    // Include question functions
-    var question = require('./question');
-    // Start timer on top of the page (total time of the learning session in seconds)
-    var timer = require('./timer');
-    timer.startTimer(600); // TODO: Magic number 600; in the future, user will choose time in main menu.
+define(['jquery', 'app/database', 'app/messages', 'app/question', 'app/timer'], function ($, db, messages, question, timer) {
+  timer.startTimer(600); // TODO: Magic number 600; in the future, user will choose time in main menu.
 
-    // Disable autocomplete that provides suggestions when typing words
-    $('input').attr('autocomplete', 'off');
+  // Disable autocomplete that provides suggestions when typing words
+  $('input').attr('autocomplete', 'off');
 
-    $(document).ready(function() {
-    	question.show();
-    });
+  $(document).ready(function() {
+  	question.show();
+  });
 
-    // Temporary hint button
-    $("#hintButton").click(function() {
-      messages.showHint('This is a very handy hint to answer this question that you see above.');
-    });
+  // Temporary hint button
+  $("#hintButton").click(function() {
```

Comment:
The hint system will be overhauled later, so this temporary button will disappear - it was really just for the demonstration with Hedderik.